web/ui/react-app/src/pages/graph/ExpressionExplorer.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { Modal, ModalBody, ModalHeader } from 'reactstrap';
+import './ExpressionExplorer.css'
+
+interface Props {
+  show: boolean;
+  updateShow(show: boolean): void;
+  autocompleteSections: { [key: string]: string[] };
+  insertAtCursor(value: string): void;
+}
+
+interface State { }
+
+class ExpressionExplorer extends Component<Props, State> {
+  handleQueryClick = (query: string) => {
+    this.props.insertAtCursor(query);
+    this.props.updateShow(false);
+  }
+
+  toggle = () => {
+    this.props.updateShow(!this.props.show)
+  }
+
+  render() {
+    return (
+      <Modal isOpen={this.props.show} toggle={this.toggle} className="expression-explorer">
```

Comment:
`.modal-dialog` has a `max-width` of `500px`, causing long metric names to overflow:

![overflow](https://user-images.githubusercontent.com/538008/105818231-f12ee100-5fb6-11eb-85ff-d2e5f1c8531e.png)

Maybe unset the max width for the expression-explorer modal and just make it fit any content instead?

Reply:
I increased the max-width of the modal and added `overflow-wrap: break-word` to break the line for very long lines:

![image](https://user-images.githubusercontent.com/20486366/106376411-5ad33480-6395-11eb-9f50-d3a75b70f564.png)